tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -38,6 +38,9 @@ describe("Feature: Hub and Spoke", () => {
       await expect(await $(HubPage.summaryRowState("employment-section")).getText()).toBe("Not started");
       await expect(await $(HubPage.summaryRowState("accommodation-section")).getText()).toBe("Not started");
       await expect(await $(HubPage.summaryRowState("household-section")).getText()).toBe("Not started");
+      await expect(await $(HubPage.summaryRowLink("employment-section")).getHTML()).toContain("Start section: Employment");
+      await expect(await $(HubPage.summaryRowLink("accommodation-section")).getHTML()).toContain("Start section: Accommodation");
+      await expect(await $(HubPage.summaryRowLink("household-section")).getHTML()).toContain("Start section: Household residents");
```

Comment:
The changes look good, except there are some situations (like the one above) when these checks shouldn't be part of the test or test description should change. For example here we assess the state of a section. Also, we could possibly narrow it down and use different method instead of `.getHTML`. Can we use anything more specific from the list [here](https://webdriver.io/docs/api/element/)?

Reply:
For example `When a user first views the Hub, The Hub should be in a continue state` checks if the hub is in continue state but also some other things.